scheds/rust/scx_layered/src/main.rs

```diff
@@ -1833,8 +1833,15 @@ impl<'a> Scheduler<'a> {
             }
         }
 
-        skel.maps.rodata_data.ext_sched_class_addr = get_kallsyms_addr("ext_sched_class")?;
-        skel.maps.rodata_data.idle_sched_class_addr = get_kallsyms_addr("idle_sched_class")?;
+        let ext_sched_class_addr = get_kallsyms_addr("ext_sched_class");
+        let idle_sched_class_addr = get_kallsyms_addr("idle_sched_class");
+
+        if ext_sched_class_addr.is_ok() && idle_sched_class_addr.is_ok() {
+            skel.maps.rodata_data.ext_sched_class_addr = ext_sched_class_addr.unwrap();
+            skel.maps.rodata_data.idle_sched_class_addr = idle_sched_class_addr.unwrap();
+        } else {
+            warn!("skip_preempt is not supported, ignoring");
```

Comment:
Maybe mention that the addresses couldn't be read from /proc/kallsyms so that users can hunt down what's not working more easily?

Reply:
I'll add this, but also like, do we think these symbols could be exported kernel side such that we don't need to do this in the future or no?

Reply:
It will probably remain this way.

Reply:
will make this into a helper in common then, thanks!

Reply:
Oh, let's keep it in layered for now. We can move it to utils when there are other users.